msteams-platform/sbs-gs-spfx.yml

```diff
@@ -166,33 +165,33 @@ items:
 
     1. Deselect **Tab**, select **Tab(SPFx)**, and select **OK**.
 
-        :::image type="content" source="../msteams-platform/assets/images/teams-toolkit-v2/create-project-capabilities.png" alt-text="Screenshot showing how to add capabilities to your new app.":::
+        :::image type="content" source="~/assets/images/teams-toolkit-v2/spfxTab-spfx.png" alt-text="tab spfx new" border="true":::
```

Comment:
Found another one: looks like this is still pointing to the previous png file, not the latest one that I sent to you?